tests/lava/proc/monitor/test_monitors.py

```diff
@@ -342,6 +342,141 @@ def test_proc_params_accessible_in_proc_model(self):
         self.assertEqual(next(iter(exe.py_builders)).proc_params,
                          monitor.proc_params)
 
+    # New API unittests
+    def test_single_recorder_with_single_targer(self):
+        """Check if a single Recorder can monitor the spike output of a
+        neuron"""
+        shape = (1,)
+        num_steps = 6
+        neuron = LIF(shape=shape,
+                     vth=3,
+                     bias=1)
+
+        recorder = Recorder()
+        recorder.probe(neuron.s_out)
+
+        neuron.run(condition=RunSteps(num_steps=num_steps),
+                   run_cfg=Loihi1SimCfg())
+
+        spike_data = recorder.get_data(neuron.s_out)
+        self.assertTrue(np.all(spike_data == np.array([[0, 0, 1, 0, 0, 1]]).T))
+
+    def test_single_recorder_with_multiple_targets_from_single_proc(self):
+        """Check if a single Recorder can monitor multiples targets,
+        here the spike output and voltage of a neuron"""
+        shape = (1,)
+        num_steps = 6
+        neuron = LIF(shape=shape,
+                     vth=3,
+                     bias=1)
+
+        recorder = Recorder()
+        recorder.probe([neuron.s_out, neuron.v])
```

Comment:
I think we decided to not allow this anymore. There is an issue with specifying multiple sources and then also being able to specify `interval` and `offset` for them. Unless we are only supporting this kind of method call with multiple sources that all have the same `interval` and `offset`? If so, we should specify that here, maybe even put it in a comment.

Reply:
Yes I agree we don't need allow multi-source monitoring, it complicates the implementation unnecessarily.